test/signalrclient-testhost/Connections/HubConnection.cs

```diff
@@ -26,5 +26,24 @@ public string ReturnMessage(string message)
         {
             return message;
         }
+
+        public async Task InvokeWithProgress(IProgress<int> progress)
+        {
+            for (int i = 0; i < 10; i++)
+            {
+                await Task.Delay(150);
+                progress.Report(i);
+            }
+        }
+
+        public async Task<string> InvokeWithProgress(string jobName, IProgress<int> progress)
+        {
+            for (int i = 0; i < 10; i++)
+            {
+                await Task.Delay(150);
```

Comment:
same as above - these 2 tests now take 3 secs which is probably longer than all other tests combined.